resources/data/utility_bills/sdr_rates/State.tsv

```diff
@@ -0,0 +1,50 @@
+State	electricity_fixed_charge	electricity_marginal_rate	natural_gas_fixed_charge	natural_gas_marginal_rate	propane_fixed_charge	propane_marginal_rate	fuel_oil_fixed_charge	fuel_oil_marginal_rate	wood_fixed_charge	wood_marginal_rate
```

Comment:
Ugh. Missed this in the original add of this file. These electricity columns should be "elec_fixed_charge" and "elec_marginal_rate", otherwise these fields will be ignored and instead populated with OS-HPXML defaults.

@afontani @jbrossman